content/the-ember-times-issue-200.md

```diff
@@ -41,15 +42,35 @@ EmberConf 2022 was remote, so all [the videos](https://2022.emberconf.com/talks)
 
 ---
 
-## [🐹 4. Section title in sentence case](section-url)
+## [🧵 4. Embroider is legit](https://github.com/embroider-build/embroider)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Embroider has been _declared_ stable since v1.0.0, or around 6 months ago (released mid January 2021).
```

Comment:
sorry I'm not a good writer either.  But I feel like before we say `Embroider has been declared stable since v1.0.0`

We should add a 1 or 2 liner about what is Embroider? 

Such as :

`Embroider is the 2nd generation build pipeline to Ember Apps that enables the usage of “Standard JavaScript” and leverages standard JavaScript packagers such as Webpack.`

Reply:
@NullVoxPopuli thanks for the writeup, much appreciated! I applied @angelayanpan's edits in a new commit and did a few copy edits for the Times team